redi-connect-infrastructure/README.md

```diff
@@ -79,6 +79,13 @@ Run the az acr show command to retrieve credentials for the registry:
 az acr credential show --resource-group AppSvc-DockerTutorial-rg --name <registry-name>
 ```
 
+## Caveats on terraform code
+
+### custom cdn endpoints are not supported
+
+https://github.com/terraform-providers/terraform-provider-azurerm/issues/398
```

Comment:
Perhaps use a bullet item here, and improve the phrasing a bit? Reads more like scratchpad comments at the moment. Why is a terraform-providers issue being references? Perhaps introduce the link?